index.js

```diff
@@ -0,0 +1,153 @@
+
+// Displays mocked CRM Info.
+function displayCRMInfo (id, location, status) {
+  const idElement = document.getElementById("id");
+  const locationElement = document.getElementById("location");
+  const statusElement = document.getElementById("status");
+
+  idElement.innerHTML = id;
+  locationElement.innerHTML = location;
+  statusElement.innerHTML = status;
+}
+
+// Displays the mocked CRM notes.
+function displayNotes(notes) {
+  // Reset the Notes column to make room for the newly found notes.
+  clearNotes();
+
+  // Add each Note to the Notes Column object.
+  notes.forEach(note => {
+    displayNote(note);
+  });
+}
+
+// Removes the currently displayed Notes.
+function clearNotes() {
+  const noteColumns = document.getElementById("notes");
+  noteColumns.innerHTML = "";
+}
+
+// Set the tab to Info and hide Notes.
+function showInfo() {
+  const infoButton = document.getElementById("infoButton");
+  const notesButton = document.getElementById("notesButton");
+  infoButton.classList.add('selected');
+  notesButton.classList.remove('selected');
+
+  const infoSection = document.getElementById("infoSection");
+  infoSection.classList.remove("displayNone");
+  const notesSection = document.getElementById("notesSection");
+  notesSection.classList.add("displayNone");
+}
+
+// Set the tab to Notes and hide Info.
+function showNotes() {
+  const infoButton = document.getElementById("infoButton");
+  const notesButton = document.getElementById("notesButton");
+  infoButton.classList.remove('selected');
+  notesButton.classList.add('selected');
+
+  const infoSection = document.getElementById("infoSection");
+  infoSection.classList.add("displayNone");
+  const notesSection = document.getElementById("notesSection");
+  notesSection.classList.remove("displayNone");
+}
+
+function displayNote(note) {
+  const noteColumns = document.getElementById("notes");
+  // Build the shadowed backdrop for the Note.
+  let noteBlock = document.createElement("div");
+  noteBlock.classList.add("noteBlock");
+
+  // Build the Header of the note containing the author and the time written.
+  let noteHeader = document.createElement("p");
+  noteHeader.classList.add("row");
+
+  let noteHeaderAuthor = document.createElement("div");
+  noteHeaderAuthor.innerHTML = note.author;
+  noteHeaderAuthor.classList.add("font", "noteKey");
+
+  let noteHeaderTime = document.createElement("div");
+  noteHeaderTime.innerHTML = note.time;
+  noteHeaderTime.classList.add("font", "noteValue");
+
+  noteHeader.appendChild(noteHeaderAuthor);
+  noteHeader.appendChild(noteHeaderTime);
+
+  // Build the blurb of the note;
+  let noteBlurb = document.createElement("p");
+  let noteBlurbText = document.createTextNode(note.blurb);
+  noteBlurb.classList.add("row", "font");
+  noteBlurb.appendChild(noteBlurbText);
+
+  // Append the Header and the Blurb to the Note block.
+  noteBlock.appendChild(noteHeader);
+  noteBlock.appendChild(noteBlurb);
+  noteColumns.appendChild(noteBlock);
+}
```

Comment:
feels kinda messy but again deferring to xav because I dont have ideas for best practices here. 

I do understand it though, which is really the purpose of my review, so that's good.